web/src/classic/components/organisms/Settings/ProjectList/index.tsx

```diff
@@ -63,12 +68,18 @@ const ProjectList: React.FC<Props> = ({ workspaceId }) => {
           ),
         }}
       </TabSection>
+      <ProjectTypeSelectionModal
```

Comment:
Please just add the contents of this modal to the `ProjectCreationModal`. It simplifies the code, and avoids the stutter that happens now when you select a project type (the first modal closing and the second modal opening)

Reply:
isn't the code like this more separated so when we back to beta all projects will be beta and the only work we need to d is remove the new modal , and for now it works in sequence so even if the user close without choosing the default will be classic and the original creation modal will open immediately